src/features/game/events/landExpansion/plant.test.ts

```diff
@@ -397,7 +397,79 @@ describe("plant", () => {
 
     expect(plantedAt).toBe(dateNow - parnsipTime * 0.5);
   });
+  it("yields 20% more parsnip if bumpkin is equipped with Parsnip Tool", () => {
+    const PARSNIP_STATE: GameState = {
+      ...TEST_FARM,
+      bumpkin: {
+        ...INITIAL_BUMPKIN,
+        equipped: { ...INITIAL_BUMPKIN.equipped, tool: "Parsnip" },
+      },
+    };
+
+    const state = plant({
+      state: {
+        ...PARSNIP_STATE,
+        inventory: {
+          "Parsnip Seed": new Decimal(1),
+          // "Mysterious Parsnip": new Decimal(1),
```

Comment:
We might not need this commented line here.

Reply:
Fixed.